.buildkite/test-sample-yamls.yml

```diff
@@ -59,3 +59,21 @@
     - ./.buildkite/setup-env.sh
     # Use KubeRay operator image from the latest release
     - source .venv/bin/activate && BUILDKITE_ENV=true OPERATOR_IMAGE=quay.io/kuberay/operator:v1.1.0 python3 tests/test_sample_rayservice_yamls.py
+
+- label: 'Test Autoscaler E2E (nightly operator)'
+  instance_size: large
+  image: golang:1.20
+  commands:
+    - source .buildkite/setup-env.sh
+    - kind create cluster --wait 900s --config ./tests/framework/config/kind-config-buildkite.yml
+    - kubectl config set clusters.kind-kind.server https://docker:6443
+    # Build nightly KubeRay operator image
+    - pushd ray-operator
+    - IMG=kuberay/operator:nightly make docker-image
+    - kind load docker-image kuberay/operator:nightly
+    - IMG=kuberay/operator:nightly make deploy
+    - kubectl wait --timeout=90s --for=condition=Available=true deployment -n ray-system kuberay-operator
+    # Run e2e tests
+    - KUBERAY_TEST_TIMEOUT_SHORT=1m KUBERAY_TEST_TIMEOUT_MEDIUM=5m KUBERAY_TEST_TIMEOUT_LONG=10m go test -timeout 30m -v ./test/e2eautoscaler
```

Comment:
Why do we need to set `KUBERAY_TEST_TIMEOUT_XXXX`?

Reply:
To keep the same settings of e2e tests on Github Actions: https://github.com/ray-project/kuberay/blob/51b64f6f1ff757740347400b7677d4028498e291/.github/workflows/e2e-tests.yaml#L68-L72